deploy/helm/flightctl/templates/flightctl-kv-secret.yaml

```
@@ -0,0 +1,23 @@
+{{- $namespaces := list .Release.Namespace }}
+{{- if .Values.global.internalNamespace }}
+{{- $namespaces = append $namespaces .Values.global.internalNamespace }}
+{{- end }}
+{{- $existingSecret := (lookup "v1" "Secret" .Release.Namespace "flightctl-kv-secret") }}
+{{- $password := "" }}
+{{- if $existingSecret }}
+  {{- $password = (index $existingSecret.data "password") }}
+{{- else }}
+  {{- $passwordRaw := (randAlphaNum 20) }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)